PhysicsTools/TensorFlow/BuildFile.xml

```diff
@@ -0,0 +1,9 @@
+<use name="FWCore/Framework" />
+
+<use name="tensorflow-c" />
+
+<Flags CXXFLAGS="-g -O0 -fpermissive" />
```

Comment:
```-g ``` will need to be removed.
Are the other flags required due to compatibility requirements with tensorflow-c ?
If not, please drop the CXX flag modifiers